quill-cassandra/src/test/scala/io/getquill/source/cassandra/EncodingSpec.scala

```diff
@@ -0,0 +1,125 @@
+package io.getquill.source.cassandra
+
+import io.getquill._
+import java.util.Date
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class EncodingSpec extends Spec {
```

Comment:
I' having troubles to make these tests pass. Basically, two rows are inserted but only one is returned. Do you I need to set consistency levels even for a simple case like this?

I've also tried to make individual inserts (see the async test), but it doesn't work as well.